internal/commands/sbomtest/summary.go

```diff
@@ -0,0 +1,43 @@
+package sbomtest
+
+import (
+	"encoding/json"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/content_type"
+	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
+
+	"github.com/snyk/cli-extension-sbom/internal/snykclient"
+)
+
+func BuildTestSummary(resultsSummary *snykclient.SBOMTestSummary) (data []byte, contentType string, err error) {
+	summary := json_schemas.TestSummary{
+		Type: "sbom",
+		Results: []json_schemas.TestSummaryResult{
+			{
+				Severity: "critical",
```

Comment:
Maybe make use of `severities.LowSeverity` ?

Reply:
Seems like our severities constants are uppercase, but the [CLI expects](https://github.com/snyk/go-application-framework/pull/138/files) an enum of lowercase severities for now, though they discuss in the PR to maybe changing it to a regular `string` in the future